doc/man1/flux-keygen.rst

```diff
@@ -6,7 +6,7 @@ flux-keygen(1)
 SYNOPSIS
 ========
 
-**flux** **keygen** [*--name=NAME*] *PATH*
+**flux** **keygen** [*--meta=KEY=VAL...*] [*--name=NAME*] *PATH*
```

Comment:
anal retentive nit, you list meta first here, but below under options you list name first :-)

Reply:
It's only fair, you deal with my anal retentive nits all the time :-)